ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartOptionsEditor.tsx

```diff
@@ -103,8 +103,8 @@ export function TimeSeriesChartOptionsEditor(props: TimeSeriesChartOptionsEditor
         Add Query
       </Button>
       {queries.map((query, i) => (
-        <Stack key={i}>
-          <Stack direction="row">
+        <Stack key={i} spacing={1}>
+          <Stack direction="row" borderBottom={1} borderColor="grey.300">
```

Comment:
did you check, it's visible in the dark mode ? :)

Reply:
Ya, the grey.x colors are all safe, they are mapped to the right value in each mode

Reply:
<img width="1512" alt="Screen Shot 2022-10-20 at 3 19 29 PM" src="https://user-images.githubusercontent.com/2584129/197069839-a877046e-9a89-4941-8b35-41d0f08ba587.png">

Reply:
perfect !